src/dom/index.js

```diff
@@ -64,7 +64,10 @@ export function setAccessor(node, name, old, value, isSvg) {
 		}
 		l[name] = value;
 	}
-	else if (name!=='list' && name!=='type' && !isSvg && name in node) {
+	else if (typeof value === 'string' && !isSvg && name in node) {
```

Comment:
Think it'd be possible to invert this conditional and have String fall through to the complete attribute add/update/remove flow on line 75?